crates/re_types/src/datatypes/tensor_data_ext.rs

```diff
@@ -864,6 +875,11 @@ fn test_image_height_width_channels() {
         // h=1, w=1, RGB:
         (vec![1, 1, 3], Some([1, 1, 3])),
         (vec![1, 1, 1, 3], Some([1, 1, 3])),
+        (vec![1, 1, 3, 1], Some([1, 1, 3])),
```

Comment:
same here - this kind of implies that extraneous trailing ones are more common than leading ones